src/webui/src/components/overview/config/TrialConfigPanel.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { Stack, Panel, Pivot, PivotItem, PrimaryButton } from '@fluentui/react';
+import { EXPERIMENT } from '../../../static/datamodel';
+import MonacoEditor from 'react-monaco-editor';
+import { MONACO } from '../../../static/const';
+import { convertDuration } from '../../../static/function';
+import { prettyStringify } from '../../../static/json_util';
+import lodash from 'lodash';
+import '../../../static/style/logDrawer.scss';
+
+interface LogDrawerProps {
+    hideConfigPanel: () => void;
+    activeTab?: string;
+}
+
+interface LogDrawerState {
+    logDrawerHeight: number;
+}
+
+class TrialConfigPanel extends React.Component<LogDrawerProps, LogDrawerState> {
+    constructor(props: LogDrawerProps) {
+        super(props);
+
+        this.state = {
+            logDrawerHeight: window.innerHeight - 48
+        };
+    }
+
+    setLogDrawerHeight = (): void => {
+        this.setState(() => ({ logDrawerHeight: window.innerHeight - 48 }));
+    };
+
+    async componentDidMount(): Promise<void> {
+        window.addEventListener('resize', this.setLogDrawerHeight);
+    }
+
+    componentWillUnmount(): void {
+        window.removeEventListener('resize', this.setLogDrawerHeight);
+    }
+
+    render(): React.ReactNode {
+        const { hideConfigPanel, activeTab } = this.props;
+        const { logDrawerHeight } = this.state;
+        const blacklist = [
+            'id',
+            'logDir',
+            'startTime',
+            'endTime',
+            'experimentName',
+            'searchSpace',
+            'trainingServicePlatform'
+        ];
+        const filter = (key: string, val: any): any => {
+            return blacklist.includes(key) ? undefined : val;
+        };
+        const profile = lodash.cloneDeep(EXPERIMENT.profile);
+        profile.execDuration = convertDuration(profile.execDuration);
+        profile.params.maxExecDuration = convertDuration(profile.params.maxExecDuration);
+        const showProfile = JSON.stringify(profile, filter, 2);
+        return (
+            <Stack>
+                <Panel
+                    isOpen={true}
+                    hasCloseButton={false}
+                    isFooterAtBottom={true}
+                    isLightDismiss={true}
+                    onLightDismissClick={hideConfigPanel}
+                >
+                    <div className='log-tab-body'>
+                        <Pivot initialSelectedKey={activeTab} style={{ minHeight: 190, paddingTop: '16px' }}>
+                            <PivotItem headerText='Search space' itemKey='search space'>
+                                <MonacoEditor
+                                    height={logDrawerHeight - 92 - 45}
```

Comment:
why height ` - 92 - 45` ?

Reply:
45px is button's margin-top. MonacoEditor's height = window.innerHeight - (these margin values)